docs/tickets/582/plan.md

```
@@ -0,0 +1,338 @@
+const location = await getLocationByProvenanceLocationId(
+  request.provenance,
+  request.court_id,
+  listTypeLocationType
+);
+```
+
+## File Structure
+
+All changes are within existing modules. No new `libs/` modules are created.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)